### PR TITLE
document.uniqueID always returns unique id

### LINKED ISSUE #1440: "click" event handler accumulate after activation in IE11
Code:

``` js
this._events(Button).on('click', function () {

}, this);
```

After `n`'th click we have `n+1` click handlers.

### PR DESCRIPTION
**Pull request by vsesh**
Fix #1440

### CODE REVIEW

**Review by veged — CHANGES_REQUESTED** [review]
this block supposed to use in "vanilla.js" runtime that lacks `document` object — so let's do the check in more safe cross-engine way

File: common.blocks/identify/identify.vanilla.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ var counter = 0,
     },
     identify = function(obj) {
         if((typeof obj === 'object' && obj !== null) || typeof obj === 'function') {
-            var key = 'uniqueID' in obj?
+            var key = 'uniqueID' in obj && obj !== document ?
```
Comment (dfilatov) [inline common.blocks/identify/identify.vanilla.js]: I think there's a better way:

``` js
var key;
if('uniqueID' in obj) {
    obj === global.document && (obj = obj.documentElement); // global should point to this.global inside module closure
    key = 'uniqueID';
} else {
    key = expando;
}
```

Comment (dfilatov) [reply common.blocks/identify/identify.vanilla.js]: As a bonus, you won't have to check browser environment.

Comment (veged) [reply common.blocks/identify/identify.vanilla.js]: LGTM

Comment (vsesh) [reply common.blocks/identify/identify.vanilla.js]: ok

**Pull request merged**